en/docs/build-and-run/build-and-run.md

```diff
@@ -1,9 +1,16 @@
 # Build and run the program
 
-Follow the steps below to create a sample Ballerina program in VS Code.
+Follow the steps below to create a sample Ballerina program in VS Code. 
 
-<img src="https://github.com/wso2/ballerina-plugin-vscode/blob/main/resources/images/running-your-program.gif?raw=true" width="100%" />
+<img src="../img/running-your-program.gif" width="100%" />
```

Comment:
Do we really need this giff here.. IMO, it doesn't seem to serve any purpose this early. Wasn't very clear what it does.
Shall we apply two separate giffs to Code View and Diagram View instead? wdyt?